changelog.dd

```diff
@@ -183,6 +184,17 @@ $(LI $(LNAME2 aa-keyvalue, $(D byKeyValue) was added.)
 )
 )
 
+$(LI $(LNAME2 logger, An experimental logging system was added.)
+
+------
+import std.experimental.logger;
+
+void main()
+{
+    log("message logging in D");
+}
+------
+
```

Comment:
You're missing a closing paren for the LI, like everyone else who submitted a changelog entry ;).